### ISSUE TITLE
ref is not supporting

### ISSUE THREAD

**Issue by qkreltms** [llorentegerman/simple-flexbox#8]
I've tried to get ```offsetWidth``` from Column's ref.current, but it prints ```undefined```.
We need to support ref.

It's not works
```
      <Column  ref={this.thisComponent}>
        <div>test</div>
        <div>test2</div>
      </Column>
```

It works
```
      <div ref={this.thisComponent}>
        <div>test</div>
        <div>test2</div>
      </div>
```

By the way material-ui supports ref this way although it uses hook
https://github.com/mui-org/material-ui/blob/5301c24c3be1023efcec7e33bd7e7f030a87c0cb/packages/material-ui/src/Typography/Typography.js#L122-L163
```
const Typography = React.forwardRef(function Typography(props, ref) {
  const {
    align = 'inherit',
    classes,
    className,
    color = 'initial',
    component,
    display = 'initial',
    gutterBottom = false,
    noWrap = false,
    paragraph = false,
    theme,
    variant = 'body1',
    variantMapping = defaultVariantMapping,
    ...other
  } = props;


  const Component =
    component ||
    (paragraph ? 'p' : variantMapping[variant] || defaultVariantMapping[variant]) ||
    'span';


  return (
    <Component
      className={clsx(
        classes.root,
        {
          [classes[variant]]: variant !== 'inherit',
          [classes[`color${capitalize(color)}`]]: color !== 'initial',
          [classes.noWrap]: noWrap,
          [classes.gutterBottom]: gutterBottom,
          [classes.paragraph]: paragraph,
          [classes[`align${capitalize(align)}`]]: align !== 'inherit',
          [classes[`display${capitalize(display)}`]]: display !== 'initial',
        },
        className,
      )}
      ref={ref}
      {...other}
    />
  );
});
```

**Comment by llorentegerman** [comment]
Hi, thanks, I just added the code to support ref, but I haven't merged it yet because I have to test it a little bit more.

https://github.com/llorentegerman/simple-flexbox/compare/v2.3.0?expand=1

the correct way to use it is:
```
<Row componentRef={(ref) => this. thisComponent = ref }>
    <span>content</span>
</Row>
```

`console.log(this. thisComponent.offsetWidth);`

**Comment by qkreltms** [comment]
Thanks, I'll also test it

**Comment by llorentegerman** [comment]
Included in v2.3.0 - PR #11
